hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -128,7 +128,8 @@ protected void changeSupply(
             validateTrue(token.maxSupply() >= newTotalSupply, TOKEN_MAX_SUPPLY_REACHED);
         }
 
-        final var treasuryAccount = accountStore.get(asAccount(treasuryRel.accountNumber()));
+        final var treasuryAccount =
+                accountStore.get(asAccount(treasuryRel.accountId().accountNum()));
```

Comment:
Why are we wrapping this in asAccount if it's already an account ID?
Couldn't this just be
`accountStore.get(treasuryRel.accountId());`

Reply:
Updated, thanks.